build.yaml

```diff
@@ -1,4 +1,4 @@
-# vim: ai ts=4 sts=4 et sw=4 ft=yaml fdm=indent et foldlevel=0
+# vim: ai ts=4 sts=4 et sw=4 ft=yaml et
```

Comment:
This is unrelated, but a pain every time I open one of these files. Personal preferences (display folding) shouldn't be hard-coded into files.